engine/src/main/java/org/hibernate/validator/internal/cfg/context/ConstraintDefinitionContextImpl.java

```diff
@@ -32,7 +32,7 @@
 
 	private boolean includeExistingValidators = true;
 
-	private final Set<Class<? extends ConstraintValidator<A, ?>>> validatorTypes = newHashSet();
+	private final Set<ConstraintValidatorDescriptor<A>> validatorTypes = new HashSet<>();
```

Comment:
I don't think `validatorTypes` is accurate as the descriptor also contains the lambda in the lambda case. Why not `validatorDescriptors`?